src/nix/copy.cc

```diff
@@ -67,6 +67,10 @@ struct CmdCopy : StorePathsCommand
                 "To copy a closure from another machine via SSH:",
                 "nix copy --from ssh://server /nix/store/a6cnl93nk1wxnq84brbbwr6hxw9gp2w9-blender-2.79-rc2"
             },
+            Example{
```

Comment:
There should probably be an `#ifdef ENABLE_S3` around this.

Reply:
Fixed!